Lib/fontmake/__main__.py

```diff
@@ -137,7 +137,7 @@ def parse_mutually_exclusive_inputs(parser, args):
         ):
             ufo_paths.append(filename)
         else:
-            parser.error(f"Unknown input file extension: '{filename}'")
```

Comment:
don't need to use `.format()`, you can keep the f-string and add `!r` to get the repr, e.g. `f"{filename!r}"`

Reply:
I see thanks, I followed what flake8 said without much thinking, and did not know !r was available as a modifier to  another {variable}. Fixed, pushing shortly.